packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
```

Comment:
We could explain a bit more that this is done automagically and the user doesn't do anything. Might be misleading though, because costs will obv. increase....not sure if it makes sense.

```suggestion
- Stream enables us to automatically scale to millions of viewers (no actions required from your side)
```

Reply:
I think this is an implicit fact. I'll take your suggestion here :)